src/utils/getMatchedPassagesFromFamiy.ts

```diff
@@ -0,0 +1,15 @@
+import { TDocumentPage, TMatchedFamily, TPassage } from "@types";
+
+export const getMatchedPassagesFromSearch = (families: TMatchedFamily[], document: TDocumentPage): [TPassage[], number] => {
+  let passageMatches: TPassage[] = [];
+  let totalNoOfMatches = 0;
+  families.forEach((family) => {
+    family.family_documents.forEach((cacheDoc) => {
+      if (document.slug === cacheDoc.document_slug) {
+        passageMatches.push(...cacheDoc.document_passage_matches);
+        totalNoOfMatches = family.total_passage_hits;
```

Comment:
question: should this just be `=` and not an additive operator?

Reply:
Yes, we only want to use the value provided for the given doc we are currently viewing.

Reply:
Ah - then would it make more sense [here](https://github.com/climatepolicyradar/navigator-frontend/pull/395/files#r1979044770)?

Reply:
Actually scrap my whole thought - I get what this is doing. It's finding the document 
in the family and then hoisting `family.total_passage_hits` on that match.